src/ReachSets/ContinuousPost/GLGM06/post.jl

```diff
@@ -0,0 +1,47 @@
+function post(𝒜::GLGM06,
+              𝑃::InitialValueProblem{<:AbstractContinuousSystem},
+              𝑂::Options)::ReachSolution{Zonotope}
+
+    # ==================================
+    # Initialization and discretization
+    # ==================================
+
+    𝑂 = merge(𝒜.options.defaults, merge(𝑂, 𝒜.options.specified))
+    max_order = 𝑂[:max_order]
+    δ, T = 𝑂[:δ], 𝑂[:T]
+    N = round(Int, T / δ)
+
+    # compute and unrwap discretized system
+    𝑃_discrete = discretize(𝑃, δ, algorithm=𝑂[:discretization], set_operations="zonotope")
```

Comment:
You do not pass on two of the discretization options that are defined in `options_GLGM06`. Either remove them or pass them here.